scripts/make_fake_pm_data.js

```diff
@@ -0,0 +1,134 @@
+        dataGreenhouseGas.push(building.GREENHOUSE_GAS_EMISSIONS);
+    })
+
+    let meanEUI = Math.round((sumEUI / dataset.length))
+    let meanEnergyStar = Math.round((sumEnergyStar / dataset.length))
+    let meanGreenhouseGas = Math.round((sumGreenhouseGas / dataset.length))
+
+    let stdDevEUI = standardDeviation(meanEUI, dataEUI, dataset.length)
+    let stdDevEnergyStar = standardDeviation(meanEnergyStar, dataEnergyStar, dataset.length)
+    let stdDevGreenhouseGas = standardDeviation(meanGreenhouseGas, dataGreenhouseGas, dataset.length)
+
+    let normalizedBuildingsObj = dataset.map(building => {
+        return {
+            ...building,
+            latitude: shiftCoordinates(building.LAT_LONG[0], 3),
+            longitude: shiftCoordinates(building.LAT_LONG[1], -3),
+            city: "Springfield",
+            year: 2016,
+            reported_address: building.PROPERTY_ADDRESS,
+            property_type: building.PRIMARY_USE,
+            property_name: building.NOTES,
+            numunits: randomNumber(1, 100),
+            numfloors: Number(building.FLOORS),
+            numbuildings: Number(building.PROPERTY_BUILDINGS),
+            yearbuilt: Number(building.YEAR_BUILT),
+            reported_gross_floor_area: Number(building.APPROX_BUILDING_AREA.split(",").join("")),
+            site_eui: (Math.sign(randomNumberOfStandardDeviations(-1, 1)) === -1) ? (Math.round(meanEUI + (randomNumberOfStandardDeviations(-1, 2) * stdDevEUI))) : (Math.round((randomNumberOfStandardDeviations(-1, 1) * stdDevEUI) + meanEUI)),
+            source_eui: (Math.sign(randomNumberOfStandardDeviations(-1, 1)) === -1) ? (Math.round(meanEUI + (randomNumberOfStandardDeviations(-1, 2) * stdDevEUI))) : (Math.round((randomNumberOfStandardDeviations(-1, 1) * stdDevEUI) + meanEUI)),
+            energy_star_score: (Math.sign(randomNumberOfStandardDeviations(-1, 1)) === -1) ? (Math.round(meanEUI + (randomNumberOfStandardDeviations(-1, 2) * stdDevEUI))) : (Math.round((randomNumberOfStandardDeviations(-1, 1) * stdDevEUI) + meanEUI)),
+            total_ghg_emissions: (Math.sign(randomNumberOfStandardDeviations(-1, 1)) === -1) ? (Math.round(meanEUI + (randomNumberOfStandardDeviations(-1, 2) * stdDevEUI))) : (Math.round((randomNumberOfStandardDeviations(-1, 1) * stdDevEUI) + meanEUI)),
+            total_ghg_emissions_intensity: (Math.sign(randomNumberOfStandardDeviations(-1, 1)) === -1) ? (Math.round(meanEUI + (randomNumberOfStandardDeviations(-1, 2) * stdDevEUI))) : (Math.round((randomNumberOfStandardDeviations(-1, 1) * stdDevEUI) + meanEUI))
+        }
+    })
+    
+    cb(null, normalizedBuildingsObj)
+}
+
+function CSVify(normalizedBuildingsObj, cb) {
+    console.log("Generating CSV")
+    const fields = _.keys(normalizedBuildingsObj[0]);
+    const csv = json2csv({ data: normalizedBuildingsObj, fields: fields });
+    cb(null, csv);
+}
+
+function writeToCsv(csv, cb) {
+    fs.writeFile(`${argv[1]}`, csv, 'utf8', (err) => {
```

Comment:
Let's just push this out via stdout with `process.write`, so that the bash/zsh syntax ends up as:

```bash
$ node make_fake_pm_data.js input_file.json > output_file.json
```

The advantage here is the same as all unix stdin/stdout ... we can pipe it to another process (eg. we want to check we're not losing lines in processing, so maybe `node make_fake_pm_data.js input_file.json | wc -l`) or we want to run it in a loop to generate 2017, 2018, 2019, 2020 for multiple cities... there's no reason to support code that uses node `fs.writeFile` when the shell facility is more flexible.